ethdb/database_test.go

```diff
@@ -49,15 +49,18 @@ var test_values = []string{"", "a", "1251", "\x00123\x00"}
 func TestLDB_PutGet(t *testing.T) {
 	db, remove := newTestLDB()
 	defer remove()
+	//enable metrics
+	metrics.Enabled = true
```

Comment:
This is racy because some tests that read the global variable can be running in parallel.